ui/app/AppLayouts/Chat/ChatColumn/EmptyChat.qml

```diff
@@ -17,38 +17,79 @@ Item {
         source: "../../../../onboarding/img/chat@2x.jpg"
     }
 
-    StyledText {
-        text: `<a href="shareKey" style="color:${Style.current.blue};text-decoration:none;">` +
-              //% "Share your chat key"
-              qsTrId("share-your-chat-key")
-              + '</a> ' +
-              //% "or"
-              qsTrId("or")
-              + ` <a href="invite" style="color:${Style.current.blue};text-decoration:none;">` +
-              //% "invite"
-              qsTrId("invite")
-              + '</a> ' +
-              //% "friends to start messaging in Status"
-              qsTrId("friends-to-start-messaging-in-status")
-        textFormat: Text.RichText
-        wrapMode: Text.WordWrap
-        horizontalAlignment: Text.AlignHCenter
-        anchors.right: walkieTalkieImage.right
-        anchors.left: walkieTalkieImage.left
+    Item {
+        id: links
         anchors.top: walkieTalkieImage.bottom
-        font.pixelSize: 15
-        color: Style.current.darkGrey
-        onLinkActivated: function (linkClicked) {
-            switch (linkClicked) {
-                case "shareKey":
```

Comment:
I wish there was a way to add a `text-decoration: underline` to a hovered link using `onLinkHovered`, but it seems that there's no access to that.

I also tried inline styles and `onmouseover` and `onmouseout` events to alter the style of the inline link but these don't seem to be supported.

Long story short, I had to break the text up into smaller pieces to gain full control over how links behave in here.